polyphemus/lib/etls/ipi/ipi_etls.rb

```diff
@@ -33,7 +33,7 @@ def add_single_cell_linkers!
           process_watch_type_with(
             bucket('data')
               .watcher('single_cell_processed')
-              .watch(/^single_cell_[^\/]*\/processed\/((?!POOL).)*$/),
+              .watch(/^single_cell_[^\/]*\/processed\/.*\/IPI((?!POOL)[^\/])+(\/[^\/]*)?$/),
```

Comment:
The main fix: previously this would expand ANY folder underneath the IPI record folder, and cause a lot of unnecessary processing that didn't catch anything.

All relevant IPI record folders may either contain files directly in them, or one folder deep, no more.

Reply:
These allow additional characters between "IPI"/"IPIPOOL" and the next "/" correct?  I'm looking at folder structure and trying to double check what I can.